test/find_with_order_test.rb

```diff
@@ -21,6 +21,11 @@ def test_find_id_with_order
     test_order.call [3, 2, 1]
   end
 
+  def no_sql_injection
```

Comment:
Thanks for adding this test case. 

The test method should start with `test_` in minitest.
See: https://github.com/seattlerb/minitest#unit-tests
```suggestion
  def test_no_sql_injection
```